### ISSUE TITLE
Mandrill is forcing people to start paying for MailChimp or (suggested) move to SparkPost...

### ISSUE THREAD

**Issue by chrisco23** [Wylio/meteor-mandrill#18]
I've been using this package for a long time but my app is still in a sort of closed alpha state and I have no intention of paying for MailChimp right now for a few test emails.  Mandrill itself recommends now moving to SparkPost.

Anybody else hit with this recently?  Any work on a SparkPost package, or some other recommendation?

**Comment by logankoester** [comment]
:+1:

**Comment by sourabh-shankar** [comment]
Will this package work after mandrill merger with mailchimp?

**Comment by danlg** [comment]
same issue for me. any recommendation for SparkPost meteor package ?

**Comment by sourabh-shankar** [comment]
just to confirm "Wylio/meteor-mandrill" is working fine once you merge mandrill account with mailchimp. you have to pay a monthly fee though. also, you need to verify your domain with DKIM and SPF keys

**Comment by chrisco23** [comment]
Good to know.  However I've been working on an extremely-slow-moving (specs trickling in) project that will be largely a notification service but still with very low volume email, for the foreseeable future, maybe always, and I really don't like having one more nagging bill for something I don't need.  I don't even like paying $5 for Google Apps and only using 0.001% of it, but that's another story.

I've started down the path with Sparkpost, who welcomes Mandrill refugees.  They do the DKIM and SPF keys too but that takes about 2 minutes to add two TXT DNS records and verify.  But I remember I did go through a fair amount of work originally getting Mandrill working with all the reactive SSR etc.  We need to email HTML tables full of reactive data.  I don't know yet how hard the real transition will be.  I'm planning to look into both rgnevashev:sparkpost and the npm sparkpost packages.
